test/store/cards.spec.js

```diff
@@ -14,10 +14,15 @@ describe('Built-in cards', () => {
     };
 
     collection.forEach(card => {
-      const abilities = (card.abilities || []).concat(card.command || []);
-      abilities.forEach(ability => {
-        executeCmd(state, ability, dummyCurrentObj);
-      });
+      try {
+        const abilities = (card.abilities || []).concat(card.command || []);
+        abilities.forEach(ability => {
+          executeCmd(state, ability, dummyCurrentObj);
+        });
+      } catch (err) {
+        console.error(`Error executing ability for ${card.name} card!`);  // eslint-disable-line no-console
```

Comment:
Do we have this comment everywhere we have a `console.error`? Maybe we can just enable `console.error` and not `console.log` if that's possible?

Reply:
Yeah, let me try that